### ISSUE TITLE
Why server print "ERROR AdbMonitor" and "Restarting adb daemon" at first start?

### ISSUE THREAD

**Issue by svladimirs** [Genymobile/gnirehtet#124]
I set server to start with windows. Every time server print about "ERROR AdbMonitor", but after that restarts and works.
2018-07-11 12:59:44.747 INFO Main: Starting relay server...
2018-07-11 12:59:44.762 INFO Relay: Relay server started
2018-07-11 12:59:45.776 ERROR AdbMonitor: Failed to monitor adb devices: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. (os error 10061)
2018-07-11 12:59:45.776 INFO AdbMonitor: Restarting adb daemon
* daemon not running; starting now at tcp:5037
* daemon started successfully
If I close server and start again it works normal:
2018-07-11 16:06:16.844 INFO Main: Starting relay server...
2018-07-11 16:06:16.846 INFO Relay: Relay server started

**Comment by rom1v** [comment]
If you execute `gnirehtet autostart` (or `gnirehtet autorun`), it monitors device using a command to the adb daemon. Whenever the server does not respond, it (re)starts it. On windows start, unless you manually execute `adb start-server`, it is not started, so it starts it.

TL;DR: this is expected.
